scripts/install.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Something to think about later. 
Right now we have 3 places with build/run/test helpers `composer.json`(scripts), `DDMakefile`, and these scripts.

I think it would be great to limit it to at most two: I.e. any combination of `scripts + makefile`, or `scripts + composer.json`, or `composer.json + makefile`. Or maybe only `composer.json` if its powerful enough.

Reply:
Good idea! Let me try how it plays out putting everything in the composer!